### ISSUE TITLE
Feature request: option to lock naked edges

### ISSUE THREAD

**Issue by ekatzenstein** [mikolalysenko/refine-mesh#2]
Currently the smoothing doesn't work well with open meshes unless we can lock the naked edges.

**Comment by mikolalysenko** [comment]
Yeah, and eventually would like to implement it.  Also would be nice to combine with adaptive/localized refinement.

**Comment by ekatzenstein** [comment]
Agreed! Would be very cool.

**Comment by mikolalysenko** [comment]
If you're interested in working on this stuff, we could hop on IRC and sketch out some ideas.  I would really love to get the mesh refinement working in nearly real time so we could do advanced mesh editing stuff in browser (like mesh mixer/laplacian shape editing).

**Comment by ekatzenstein** [comment]
Sure, I'd love to chat about it. What time works for you?

**Comment by mikolalysenko** [comment]
Let's do it right now!  If you want to talk on freenode I'm mikolalysenko, and @mikolalysenko on twitter.

**Comment by ekatzenstein** [comment]
OK sounds good. I'm new to freenode but I'll give it a shot. I'm in the #stackgl channel now
